infra/base-images/base-builder/Dockerfile

```diff
@@ -148,6 +148,7 @@ RUN mkdir $PRECOMPILED_DIR
 WORKDIR $SRC
 
 RUN git clone -b stable https://github.com/google/AFL.git afl
```

Comment:
Remove afl here if it's no longer being used?

Reply:
It is still used for the afl driver - https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-builder/compile_afl#L27. Once this stablizes, can try use https://github.com/AFLplusplus/AFLplusplus/blob/068bef5eab942df0a133c92522f2ab81b28ac636/utils/aflpp_driver/aflpp_driver.c. However, this are still some differences that i would want to consult @jonathanmetzman and Marc with (e.g. this code is different from llvm version - https://github.com/AFLplusplus/AFLplusplus/blob/068bef5eab942df0a133c92522f2ab81b28ac636/utils/aflpp_driver/aflpp_driver.c#L242)